lib/protocol/routes.js

```diff
@@ -103,7 +103,10 @@ const METHOD_MAP = {
   },
   '/wd/hub/session/:sessionId/window': {
     GET: {command: 'getWindowHandle'},
-    POST: {command: 'setWindow', payloadParams: {required: ['name']}},
+    POST: {command: 'setWindow', payloadParams: {
```

Comment:
really what we mean is that either name or handle may be present, but at least one _must_ be. is there a way to express that here?

Reply:
Where is the `name` coming from? The spec seems to just want a mandatory `handle`.

Reply:
I'm not sure where the `name` comes from. It just was there. I could change the current implementation and add a validation helper, which is gonna check if either parameter is set

Reply:
`name` is [JSONWP](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidwindow), `handle` is W3C

Reply:
this is basically the same situation as sendKeys

Reply:
ok, I've added makeArgs call similar to sendKeys